src/SignUp/signup.html

```diff
@@ -49,7 +49,7 @@
             </div>
             <div class="signup-credential-container">
                 <label>Password</label>
-                <input type="password" name="" id="">
+                <input type="password" placeholder="*************" name="" id="">
             </div>
             <div class="signup-rem-forgetpass-container">
                 <div>
```

Comment:
Suggestion: the classname could be kept shorter like "signup-container"
since, in the signup container, there is no remember field or forget the password. the longer name could confuse the reader and chances of error prevails.